mlocal/checks/project-post.chk

```diff
@@ -78,12 +83,14 @@ printf " checking: namespace: CLONE_NEWNS... "
 if ! printf "#define _GNU_SOURCE\n#include <sched.h>\nint main() { unshare(CLONE_NEWNS); }" | \
    $tgtcc -x c -o /dev/null - >/dev/null 2>&1; then
 	echo "no"
-	echo
-	echo "This host does not support the CLONE_NEWNS (mount) namespace flag! You"
-	echo "really really really don't want to run Singularity containers without a"
-	echo "Separate mount name namespace!"
-	echo
-	exit 2
+	if [ "$build_runtime" -eq 1 ]; then
```

Comment:
These checks are to ensure `mconfig` doesn't `exit 2` when these `runtime` related checks fail.